.github/workflows/CodeCov.yml

```diff
@@ -18,7 +18,7 @@ jobs:
     - name: Set up Julia
       uses: julia-actions/setup-julia@latest
       with:
-        version: '1.10'
+        version: '1.11'
```

Comment:
Is it worthwhile testing on both 1.10 and 1.11?

Reply:
I was thinking it would be good to test both the latest Julia version (1.11) and the last version with long-term support (1.10), since this is a fairly generic package. Having the benchmark tables for both versions might also help us catch compiler regressions.

Reply:
Agreed, it would be better to test on both.

Also, you don't need both a CodeCov and a CI action

Reply:
Ok, I've removed the CodeCov workflow and added the relevant line to the CI workflow.